rows.go

```diff
@@ -111,6 +115,14 @@ func (rows *Rows) Next() bool {
 	}
 }
 
+// GetStyleID provise a function to get the current Rows' style ID. It will
```

Comment:
I suggest changing the description of this exported function like this, and no example is needed.

```text
GetStyleID will return the style ID of the current row. The default style ID is 0.
```